src/mantine-dates/src/components/TimeInput/TimeInput.tsx

```diff
@@ -35,17 +40,38 @@ export const TimeInput = factory<TimeInputFactory>((_props, ref) => {
     props,
   });
 
+  const validTime = () => {
+    const minTime = dayjs(`2000-01-01 ${props.minTime}`);
+    const maxTime = dayjs(`2000-01-01 ${props.maxTime}`);
+    const value = dayjs(`2000-01-01 ${props.value}`);
+    let _value;
```

Comment:
Do not use let, return value if the condition matched instead.

Reply:
updated as per comment